config.yml

```diff
@@ -439,6 +439,21 @@ terraform:
         - git --version
     template_vars:
       BASE_IMAGE_VERSION: 3.8
+  "0.13.4":
```

Comment:
Hello, I think you can override the version `0.13.1` instead of creating a new one.

Reply:
Terraform new versions aren't retro-compatible. We need to maintain multiple minor versions.

Reply:
I think we can stop maitaining 0.12.29. Can you confirm @axelpavageau ?

Reply:
Ok 😢